godtools/App/AppDiContainer.swift

```diff
@@ -337,6 +337,13 @@ class AppDiContainer {
         return ShareToolScreenTutorialNumberOfViewsCache(sharedUserDefaultsCache: sharedUserDefaultsCache)
     }
     
+    func getToolLanguagesUseCase() -> GetToolLanguagesUseCase {
```

Comment:
I like having the use cases in here, but it does make me wonder how it'll work if we start using use cases more frequently, since most classes right now don't have a reference to AppDiContainer.